Dockerfile

```diff
@@ -0,0 +1,37 @@
+FROM grafana/tanka:0.15.1 as tanka
```

Comment:
This can probably be replaced by `kubectl` only.

I used Tanka in a test phase to run https://github.com/xvzf/crd-libs/blob/master/main.jsonnet but I don't think that intermediary step is necessary. @xvzf should be able to tell for sure.

Reply:
This has been a temporary patch for converting some CRDs that did not show up in the openapi v2 definitions. It seems it is resolved right now!